docs/docs/services/qix-engine/access-control.md

```diff
@@ -54,64 +143,89 @@ Change owner | Change the owner of a resource.
 Change role  | Change user role.
 Export data  | Export data from an object.
 
-## Resources
+**QUESTION: Are not the actions referred to as `create`, `read`, ..., and `export` (not "Create", "Export data" and so on)?**
 
-There are a number of resources which you may use in your rule conditions.
-All supported resources are listed below.
+##### Examples
 
-### Shared attributes
+The following expression evaluates to `true` if the action is `read` or `update`:
 
-Attribute         | Description
------------------ | -----------
-resource.id       | The unique identifier for the resource.
-resource.name     | The name of the resource, if any.
+```c
+resource._actions = {"read", "update"}
+```
 
-### User
+#### Resource-type Specific Attributes
 
-The user resource contains all attributes defined in the [JWT header used to authenticate the user](../../tutorials/authorization.md#json-web-token).
+Depending on what type of resource that is being accessed, `resource` carries different attributes
 
-Example JWT:
+If the resource type is `"App"`, `resource` has no additional attributes.
 
-```json
-{
-  "sub": "john-doe",
-  "name": "John Doe",
-  "employeeType": "developer",
-  "tags": ["research"],
-  "custom": {
-    "country": "sweden"
-  }
-}
+If the resource type is `"App.Object"`, `resource` has the following additional attributes:
+
+Attribute | Description | Example condition
```

Comment:
What about these attributes if the resource type is "App"? What happens if a rule refers to these attributes with the "wrong" resource type (App)? I think we should document such behavior.